src/app/Main.js

```diff
@@ -90,8 +90,18 @@ function runApp(initial_state) {
     }
 
     const config = initial_state.offchain.config;
+
+    const steemSelectedRpc = localStorage.getItem('steemSelectedRpc');
+
+    if (steemSelectedRpc) {
```

Comment:
Overwrites `steemSelectedRpc` with the config value even if a valid value was just loaded.
```suggestion
    if (!steemSelectedRpc) {
```

Reply:
I checked your comment.

I fixed bug and updated your comments.
Thanks for your kindly review.